### PR TITLE
Update maven and mappings

### PR DESCRIPTION
**Pull request by Grayray75**
Uses the new maven and updates yarn mappings.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,14 +37,14 @@ repositories {
 		url = "https://libraries.minecraft.net/"
 	}
 	maven {
-		name = "legacy-fabric"
-		url = "https://dl.bintray.com/legacy-fabric/Legacy-Fabric-Maven"
+		name = 'legacy-fabric'
```
Comment (hYdos) [inline build.gradle]: change the `'` to `"` please

File: src/main/java/net/fabricmc/loader/EntrypointStorage.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 package net.fabricmc.loader;
 
+import net.fabricmc.loader.EntrypointStorage.Entry;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/EntrypointStorage.java]: what happened here

File: src/main/java/net/fabricmc/loader/discovery/ModResolver.java [hunk 3]
```diff
@@ -23,6 +23,7 @@
 
 import net.fabricmc.loader.FabricLoader;
 import net.fabricmc.loader.api.metadata.ModDependency;
+import net.fabricmc.loader.discovery.ModResolver.UrlProcessAction;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/discovery/ModResolver.java]: you have left quite a few unused imports in

File: src/main/java/net/fabricmc/loader/gui/FabricMainWindow.java [hunk 4]
```diff
@@ -60,7 +60,9 @@
 import javax.swing.tree.DefaultTreeCellRenderer;
 import javax.swing.tree.DefaultTreeModel;
 import javax.swing.tree.TreeNode;
-
+import net.fabricmc.loader.gui.FabricMainWindow.CustomTreeNode;
+import net.fabricmc.loader.gui.FabricMainWindow.IconInfo;
+import net.fabricmc.loader.gui.FabricMainWindow.IconSet;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/gui/FabricMainWindow.java]: concern

File: src/main/java/net/fabricmc/loader/launch/knot/KnotClassDelegate.java [hunk 5]
```diff
@@ -20,6 +20,7 @@
 import net.fabricmc.api.EnvType;
 import net.fabricmc.loader.game.GameProvider;
 import net.fabricmc.loader.launch.common.FabricLauncherBase;
+import net.fabricmc.loader.launch.knot.KnotClassDelegate.Metadata;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/launch/knot/KnotClassDelegate.java]: 2 more after this

File: src/main/java/net/fabricmc/loader/metadata/V0ModMetadata.java [hunk 6]
```diff
@@ -33,6 +33,7 @@
 import net.fabricmc.loader.api.metadata.ModDependency;
 import net.fabricmc.loader.api.metadata.ModEnvironment;
 import net.fabricmc.loader.api.metadata.Person;
+import net.fabricmc.loader.metadata.V0ModMetadata.Mixins;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/metadata/V0ModMetadata.java]: how did this happen

File: src/main/java/net/fabricmc/loader/metadata/V1ModMetadata.java [hunk 7]
```diff
@@ -34,6 +34,8 @@
 import net.fabricmc.loader.api.metadata.ModDependency;
 import net.fabricmc.loader.api.metadata.ModEnvironment;
 import net.fabricmc.loader.api.metadata.Person;
+import net.fabricmc.loader.metadata.V1ModMetadata.IconEntry;
```
Comment (hYdos) [inline src/main/java/net/fabricmc/loader/metadata/V1ModMetadata.java]: I'm concerned

**Review by hYdos — CHANGES_REQUESTED** [review]

**Comment by Grayray75** [comment]
@hYdos All these imports where generated by gradle.
But you are right they are all unnecessary...

**Comment by BluCobalt** [comment]
Going to run workflow

**Review by BluCobalt — APPROVED** [review]

**Review by halotroop2288 — APPROVED** [review]

**Review by BoogieMonster1O1 — APPROVED** [review]

**Pull request merged**